.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  bless:
+    name: Bless
```

Comment:
"bless" seems like an odd name for what's essentially just testing, isn't it? Should the job's name just be "test" instead?

Reply:
Ah yep whoops, that's a relic from some old actions I've written. I think "test" is a better name :+1: